src/main/java/net/sacredlabyrinth/phaed/dynmap/simpleclans/DynmapSimpleClans.java

```diff
@@ -1,181 +1,158 @@
-		}
-	}
-
-	public static void info(String msg) {
-		log.log(Level.INFO, LOG_PREFIX + msg);
-	}
-
-	public static void severe(String msg) {
-		log.log(Level.SEVERE, LOG_PREFIX + msg);
-	}
-
-	public static DynmapSimpleClans getInstance() {
-		return instance;
-	}
-
-	public MarkerAPI getMarkerApi() {
-		return markerApi;
-	}
-
-	public ClanManager getClanManager() {
-		return simpleclansCore.getClanManager();
-	}
-
-	public DynmapAPI getDynmapApi() {
-		return dynmapApi;
-	}
-
-	public ClanHomes getClanHomes() {
-		return clanHomes;
-	}
-
-	public Kills getKills() {
-		return kills;
-	}
-
-	public PlayerManager getPlayerManager() {
-		return playerManager;
-	}
-
-	public SettingsManager getSimpleClansSettingsManager() {
-		return simpleclansCore.getSettingsManager();
-	}
-	
-	public String getLang(String lang) {
-		String msg = null;
-		if (lang != null) {
-			msg = getConfig().getString("language." + lang);
-		}
-		if (msg == null) {
-			msg = "Missing language for " + lang;
-		} else {
-			msg = ChatColor.translateAlternateColorCodes('&', msg);
-		}
-		return msg;
-	}
+    private static DynmapSimpleClans instance;
+    private DynmapAPI dynmapApi;
+    private MarkerAPI markerApi;
+    private SimpleClans simpleclans;
+    private HomeLayer homeLayer;
+    private KillsLayer killsLayer;
+
+    /**
+     * @return The plugin instance
+     */
+    public static DynmapSimpleClans getInstance() {
+        return instance;
+    }
+
+    /**
+     * @param messageKey The key
+     * @return Translated colored message from key
+     */
+    public static String lang(@NotNull String messageKey) {
+        String msg = instance.getConfig().getString("language." + messageKey);
+        return msg == null ?
+                String.format("Missing language for %s key", messageKey) :
+                ChatColor.translateAlternateColorCodes('&', msg);
+    }
+
+    /**
+     * Sends a debug message to the console, respecting the user decision.
+     *
+     * @param message             message to send
+     * @param respectUserDecision should the message be sent if debug is false?
+     */
+    public static void debug(String message, boolean respectUserDecision) {
+        if (respectUserDecision && !instance.getConfig().getBoolean("debug", false)) {
+            return;
+        }
+
+        instance.getLogger().info(message);
+    }
+
+    public static void debug(String message) {
+        debug(message, true);
+    }
+
+    @Override
+    public void onEnable() {
+        instance = this;
+
+        reload();
+
+        CommandManager commandManager = new CommandManager(this);
+        PluginCommand clanmap = Objects.requireNonNull(getCommand("clanmap"));
+        clanmap.setExecutor(commandManager);
+        clanmap.setTabCompleter(commandManager);
+
+        getPluginManager().registerEvents(new DynmapSimpleClansListener(this), this);
+
+        getLogger().log(Level.INFO, "Version {0} is activated!", getDescription().getVersion());
```

Comment:
Necessary?

Reply:
It was in the previous impl, I didn't change it.

Reply:
I guess it can be removed. Bukkit already prints something similar.